.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Pull Request
+
+Before you submit a pull request, please ensure you've completed the following checklist
+
+- [ ] Ensure there is an already open and relevant [GitHub issue](https://github.com/rstudio/bootstraplib/issues/new) describing the problem in detail and you've already received some indication from the maintainers that they are welcome to a contribution to fix the problem. This helps us to prevent wasting anyone's time. 
+
+- [ ] Ensure that you have signed the [individual](https://rstudioblog.files.wordpress.com/2017/05/rstudio_individual_contributor_agreement.pdf) or [corporate](https://rstudioblog.files.wordpress.com/2017/05/rstudio_corporate_contributor_agreement.pdf) contributor agreement as appropriate. You can send the signed copy to jj@rstudio.com.
```

Comment:
Ask Jeff for CLA permission to the repo and remove this.

Reply:
Oh it's actually already functional

Reply:
woot. Just need to remove then. 😄